extensions/chat/common/client.mjs

```diff
@@ -97,6 +98,15 @@ Usage: !/<Command> <arg1> <arg2> ...
       } else {
         this.helper.callC2sAPI(null, 'teleport', 5000, mapCoord);
       }
+    } else if (cmd.split(' ')[0] === '!/announce') {
```

Comment:
Optional: Maybe leave a TODO that we'll need to restrict this to admin?

Reply:
done.